PARAMETERS_MAPPING/parameters.csv

```diff
@@ -196,4 +196,17 @@ unique_id;cf_standard_name;cf_alias_name;long_name;imos_vocabulary_name;imos_voc
 201;;;hail_intensity_10min_sample;;;
 202;thickness_of_rainfall_amount;;rainfall_amount_since_9am;;;
 203;;;rainfall duration;;;
-204;rainfall_rate;;rainfall intensity_10min_sample;;;
\ No newline at end of file
+204;rainfall_rate;;rainfall intensity_10min_sample;;;
+850;number_concentration_of_biological_taxon_in_sea_water;;total abundance;Abundance of biota;;
+851;;;cell volume;Biovolume;;Mean size of cell
+852;;;length;;;Average total length
+853;mass_concentration_of_zooplankton_expressed_as_carbon_in_sea_water;;total zooplankton carbon;;;
+854;;;Shannon evenness;;;
+855;;;ratio;;;
+856;mass_concentration_of_diazotrophic_phytoplankton_expressed_as_carbon_in_sea_water;;total phytoplankton carbon;;;
+857;region;;;;;
```

Comment:
According to the CF standard name, the values for `region` must be taken from the CF standard region list. If we're unsure the values will comply with the standard region list, then I think `region` should only be defined using a long name

Reply:
Alright, changed!